config/samples/logging_v1_elasticsearch.yaml

```diff
@@ -28,6 +28,10 @@ spec:
               maxAge:   8h
         delete:
           minAge: 2d
+          pruneNamespacesInterval: 24h
+          namespaceSpec:
+          - namespace: openshift-logging
+            minAge: 5h
```

Comment:
This is a bad example actually. The Logging stack does not collect any logs from its component. This recursive collection is counter-productive. A good example would be to collect the `openshift-monitoring` logs.